### DISCUSSION TITLE
Lesson 2: 2:35

### DISCUSSION THREAD

**Discussion by TRW123123** [Cyfrin/foundry-full-course-cu#1725] [Q&A]
Hi everybody,

in the video I cant interact with the "contract" part. It says no compilied contracts and I cant change it. In the video he goes ahead and is able to choose a contract, I did not find out how it is done. thanks. 
![1](https://github.com/Cyfrin/foundry-full-course-f23/assets/169069520/8d7b98d0-4961-46e2-a1cb-df754e8afea9)
![2](https://github.com/Cyfrin/foundry-full-course-f23/assets/169069520/279fb34e-7def-4082-b6ab-e96b38629c88)

**Comment by DCVglobalnetwork** [comment]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/105791829/bffd5c6c-9119-4d56-9812-dceb92622945)

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/105791829/865f0c6c-330a-4ef4-9b2c-f1e3165edd8f)

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/105791829/47c0fa79-391e-4831-9efc-184aa8d1290e)

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/105791829/df6b3a43-dfb3-4762-b1e7-91147afb8f7c)

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/105791829/27f54d3e-83d1-45b1-a5d4-bac50c4a2737)

**Reply by DCVglobalnetwork** [reply]
youtube video you posted it

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/105791829/01dc1065-7a06-4d34-8a1f-e4232a9fa6ca)

**Comment by EngrPips** [comment]
Hello @TRW123123, You named your contract in `remix` as `simpleStorage.Sol` using the capital letter `S`  for `sol` instead of `simpleStorage.sol` with the small letter `s` for sol. Solidity compiler seems to be case-sensitive and didn't find your contract; hence, it says you didn't have any contract
